pages/TheExplorationEditor.html

```diff
@@ -123,5 +123,5 @@ <h4>
     </ol>
   </p>
 
-  <img src="static/images/hintModal.png" width="100%">
+  <img src="static/images/hintModal.png" width="100%" alt="Screenshot of the 'Hint' modal. A text area is present where content for the hint is to be entered. This text area is highlighted in red.">
```

Comment:
... 'Hint' modal. This modal contains a text area where the content for the hint is to be entered. The ...